packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/talon/clearValue2.yml

```diff
@@ -0,0 +1,28 @@
+languageId: talon
+command:
+  version: 5
+  spokenForm: clear value
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: value}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    tag: user.aaa
+    -
+    aaa: bbb
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+  marks: {}
+finalState:
+  documentContents: |-
+    tag:
```

Comment:
As above, this line should have a space at the end; see https://github.com/wenkokke/tree-sitter-talon/issues/22